src/models/users_model.js

```diff
@@ -8,9 +8,9 @@ const connect = function(next) {
 };
 
 const selectAllUsers = function(next) {
-    const query = "SELECT id, name FROM development.users";
+    const query = "SELECT * FROM development.users";
```

Comment:
won't that give the passwords of users in return? 
Is that a thing we should do?
Why do we have this function?

Reply:
We don't have a schema just yet, so its just a place holder :)